cmd/link_package.go

```diff
@@ -21,7 +21,7 @@ func LinkPackage(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interr
 	here := "../.." + strings.Repeat("/..", depth)
 
 	gopath := Gopath()
-	if len(gopath) == 0 {
+	if gopath == "" {
```

Comment:
I just think that comparing `gopath` against and empty string makes it clear that we are expecting a string from `Gopath()` instead of a more general slice.